src/parser.js

```diff
@@ -217,9 +219,10 @@ export default class Parser extends Stream {
                 this.manifest.mediaGroups || defaultMediaGroups;
 
               if (!(entry.attributes &&
-                    entry.attributes.TYPE &&
-                    entry.attributes['GROUP-ID'] &&
-                    entry.attributes.NAME)) {
+                  entry.attributes.TYPE &&
```

Comment:
Although editor indents are normally to this level, I think most of our projects line up with the open paren